Triangulation_2/include/CGAL/boost/graph/properties_2D_triangulation.h

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) 2019 GeometryFactory (France).  All rights reserved.
```

Comment:
should the header be in `internal` subdirectory to avoid direct inclusion?

Reply:
Done in 9cba48a.